Scripts/Core/EditorVR.cs

```diff
@@ -96,6 +96,7 @@ static void ResetPreferences()
 
 		void Awake()
 		{
+			ObjectUtils.hideFlags = defaultHideFlags;
```

Comment:
We talked about this live, but I'll copy here for @AndrewTHEManeri - both locations will keep the set of ObjectUtils.hideFlags (i.e. static constructor and Awake) and Andrew makes the call for PRs. I may add comments here and there, but won't create a review unless explicitly asked.